src/components/ScreenWrapper.js

```diff
@@ -35,6 +39,12 @@ const propTypes = {
     // Called when navigated Screen's transition is finished.
     onTransitionEnd: PropTypes.func,
 
+    /** Is the window width narrow, like on a mobile device */
+    isSmallScreenWidth: PropTypes.bool.isRequired,
+
+    /** The styles to pass to the KeyboardAvoidingView i.e. height/padding */
+    keyboardAvoidingViewBehavior: PropTypes.string,
```

Comment:
Can we use `PropTypes.oneOf()` here?

Reply:
Also curious if there's some general advice for when to use `'padding'` vs `'height'`? If someone wanted to create a new screen how would we explain to them which one they should use?